usecases/Concrete/Calibration/calibrationWorkflow.py

```diff
@@ -0,0 +1,159 @@
+# third party imports
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+import matplotlib as mpl
+from matplotlib import rc
+
+mpl.rcParams['font.size'] = 16
+rc('text', usetex=True)
+
+# local imports (probeye)
+from probeye.definition.inverse_problem import InverseProblem
+from probeye.definition.forward_model import ForwardModelBase
+from probeye.definition.sensor import Sensor
+from probeye.definition.likelihood_model import GaussianLikelihoodModel
+from probeye.ontology.knowledge_graph_export import export_knowledge_graph
+from probeye.ontology.knowledge_graph_import import import_parameter_samples
+from probeye.ontology.knowledge_graph_export import export_results_to_knowledge_graph
+from probeye.inference.emcee.solver import EmceeSolver
+
+# local imports (others)
+from usecases.Concrete.Prediction.three_point_bending_example import three_point_bending_example
+from utils import load_experimental_data, PosteriorPredictive
+from usecases.Concrete.Calibration.forwardmodel_linear_elastic_cylinder import LinearElasticityCylinder
+
+# =========================================
+#       Loading the experiment
+# =========================================
+# for concrete E 60 - 85 Gpa/60 - 85 10^9N/m2 /60-85 KN/mm^2
+skip_last = 145
+skip_init = 330
```

Comment:
How did you obtain these skips (in particular relevant if that should be a general function)